rules/xcodeproj.bzl

```diff
@@ -292,20 +288,20 @@ $BAZEL_INSTALLER
             else:
                 test_env_vars_dict[k] = v
 
-        scheme_action_details["environmentVariables"] = test_env_vars_dict
+        scheme_action_details_for_test["environmentVariables"] = test_env_vars_dict
```

Comment:
`test_env_vars_dict` may contain key-value pairs from `scheme_existing_envvar_overrides` that is set up on top level, i wonder whether these overrides should be just for `test` or applies all actions (right now it is just for test)

Reply:
I think you're right, it should apply to all actions. We could even make it apply to the run action and then have the test action "Use the Run action's arguments and environment variables."